samples/containers/common/build.gradle.kts

```diff
@@ -4,7 +4,6 @@ plugins {
 
 dependencies {
   implementation(project(":workflow-ui:container-common"))
-  implementation(project(":workflow-ui:core-android"))
```

Comment:
I still don't quite understand this.

This is that `.aar` dependency which was somehow in a `java-library` dependency without breaking the build.  That should be impossible.

After updating KtLint to fix its classpath issue, this dependency finally started causing problems.

```
Could not determine the dependencies of task ':samples:containers:common:compileJava'.
> Could not resolve all task dependencies for configuration ':samples:containers:common:compileClasspath'.
   > Could not resolve project :workflow-ui:core-android.
     Required by:
         project :samples:containers:common
      > No matching variant of project :workflow-ui:core-android was found. The consumer was configured to find an API of a library compatible with Java 8, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' but:
          - Variant 'debugApiElements' capability com.squareup.workflow1:core-android:1.8.0-beta02-SNAPSHOT declares an API of a library, preferably optimized for Android:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'androidJvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm'
              - Other compatible attributes:
                  - Doesn't say anything about how its dependencies are found (required its dependencies declared externally)
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)
          - Variant 'debugRuntimeElements' capability com.squareup.workflow1:core-android:1.8.0-beta02-SNAPSHOT declares a runtime of a library, preferably optimized for Android:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'androidJvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm'
              - Other compatible attributes:
                  - Doesn't say anything about how its dependencies are found (required its dependencies declared externally)
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)
          - Variant 'releaseApiElements' capability com.squareup.workflow1:core-android:1.8.0-beta02-SNAPSHOT declares an API of a library, preferably optimized for Android:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'androidJvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm'
              - Other compatible attributes:
                  - Doesn't say anything about how its dependencies are found (required its dependencies declared externally)
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)
          - Variant 'releaseRuntimeElements' capability com.squareup.workflow1:core-android:1.8.0-beta02-SNAPSHOT declares a runtime of a library, preferably optimized for Android:
              - Incompatible because this component declares a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'androidJvm' and the consumer needed a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm'
              - Other compatible attributes:
                  - Doesn't say anything about how its dependencies are found (required its dependencies declared externally)
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)
```